src/test/contact-info/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import {
+	getEditorHtml,
+	initializeEditor,
+	addBlock,
+	getBlock,
+	setupCoreBlocks,
+} from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import {
+	registerJetpackBlocks,
+	setupJetpackEditor,
+} from '../../jetpack-editor-setup';
+
+setupCoreBlocks();
+
+beforeAll( () => {
+	// Register Jetpack blocks
+	setupJetpackEditor( {
+		blogId: 1,
+		isJetpackActive: true,
+	} );
+	registerJetpackBlocks( {
+		capabilities: {
+			contactInfoBlock: true,
+		},
+	} );
+} );
+
+describe( 'Contact Info block', () => {
+	it( 'should successfully insert the block into the editor', async () => {
+		const screen = await initializeEditor();
```

Comment:
WDYT about relying upon the [`screen` import](https://callstack.github.io/react-native-testing-library/docs/api#screen) instead for this new test?

Reply:
Ah, good catch. I have to educate myself to start using this approach 😅.

Reply:
Suggestion applied in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6049/commits/6219271af4f93e170eceff26411666596d992184.